doc/help/MERGE.md

```diff
@@ -0,0 +1,44 @@
+kdb-merge(1) -- Three-way merge of KeySets.
+================================
+## DESCRIPTION
+Does a three-way merge between keysets.  
```

Comment:
I think a short introduction what a three-way merge is would be nice (explanation of ourpath theirpath basepath)